### DISCUSSION TITLE
Απορία σχετικά με το Netlify

### DISCUSSION THREAD

**Discussion by p20sami** [courses-ionio/hci#1640] [Announcements]
Στο netlify όταν συνδέω το site repository στο 2ο βήμα μου βγάζει να αγοράσω domain. Κάνω κάτι λάθος ή είναι επι πληρωμή και πρέπει να αγοράσω ενα domain?

**Comment by epidrome** [comment]
το νετλιφαη δημιουργεί σαμπ ντομεην όπως αυτό από άλλη εργασία της ομάδας μας: https://epic-hamilton-da9ac8.netlify.app/
